src/main/java/com/ldtteam/structurize/network/messages/LSStructureDisplayerMessage.java

```diff
@@ -0,0 +1,118 @@
+package com.ldtteam.structurize.network.messages;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+import net.minecraft.entity.player.EntityPlayerMP;
+import net.minecraftforge.fml.common.network.simpleimpl.IMessage;
+import net.minecraftforge.fml.common.network.simpleimpl.MessageContext;
+import org.jetbrains.annotations.NotNull;
+
+import com.ldtteam.structures.helpers.Settings;
+import com.ldtteam.structurize.Structurize;
+import com.ldtteam.structurize.client.gui.WindowBuildTool;
+import com.ldtteam.structurize.client.gui.WindowShapeTool;
+import com.ldtteam.structurize.management.linksession.ChannelsEnum;
+import com.ldtteam.structurize.management.linksession.LinkSessionManager;
+
+import java.util.Set;
+import java.util.UUID;
+
+/**
+ * Class handling the Server UUID Message.
+ */
```

Comment:
Guess wrong javadoc?

Reply:
Yeah, copy-pasta, how many times did I try to remember to change it..